src/transcript/TranscriptLanguageWithScore.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class TranscriptLanguageWithScore {
```

Comment:
A brief description in the form of block comments explaining what this class is and what does the score represents would be nice. Something similar to TranscriptEntity.ts

Reply:
Updated doc string for TranscriptLanguageWithScore